app/presenters/activity_presenter.rb

```diff
@@ -64,4 +64,9 @@ def display_title
     return "Untitled (#{id})" if title.nil?
     title
   end
+
+  def level
+    return if super.blank?
+    super.tr("_", " ")
```

Comment:
What do you think about `super.humanize(capitalize: false)` here, I love all the Rails active support stuff?

We could even use the old 'locale trick' and get the hyphen in? Something like:

````
t("activity_level.#{super}")

activity_level:
  fund: "Fund"
  …
  third_party_project: "Third-party project"
````

Reply:
Oooooh I forgot that existed!